src/librustdoc/html/static/rustdoc.css

```diff
@@ -1762,7 +1761,7 @@ div.name.expand::before {
 .type-decl > pre > .docblock.attributes.top-attr {
 	margin-left: 1.8em !important;
 }
-.type-decl > pre > .toggle-attributes {
+.type-decl > pre .toggle-attributes {
```

Comment:
What does this do?

Reply:
I don't know the meaning in this particular case, but `>` means select the direct child as opposed to allowing grandchildren to be selected (which is what ` ` does).

Reply:
It used to be a direct descendant selector, but it stopped working because of the new wrapping `docblock` element. Now it will work on all `.toggle-attributes` elements regardless of how far nested they are.

Reply:
Question: Why is there a new wrapping `docblock` element? Can that element be `<div class="type-contents-toggle"  ...>` instead of `<div class="docblock type-contents-toggle"  ...>`?